docs/source/endpoints/translations.md

```diff
@@ -125,6 +125,19 @@ This endpoint returns the proper placement for the newly created translation:
 
 ## Expansion
 
-This endpoint can be used with the {doc}`expansion` mechanism which allows getting additional information about a content item in one query, avoiding unnecessary requests.
+This service can be used with the {doc}`expansion` mechanism which allows getting additional information about a content item in one query, avoiding additional requests.
+
+As soon as plone.app.multilingual is installed, a content item is translatable.
+Which means it provides the ITranslatable interface.
+Therefore the content information provided by the content endpoint is expandable to translations.
```

Comment:
```suggestion
Translation information can be provided by the API expansion for translatable content items.
A content type is translatable if it has the plone.translatable behavior enabled. plone.app.multilingual enables this behavior for all content types at the time that it is installed.
For other content types added later it would be necessary to enable the behavior for them also.
```

Integration of @davisagli 's suggestion